packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -112,7 +114,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
 
             const miSdkApiTag = document.createElement('script');
             miSdkApiTag.setAttribute('type', 'text/javascript');
-            miSdkApiTag.setAttribute('src', 'https://app.mapsindoors.com/mapsindoors/js/sdk/4.24.7/mapsindoors-4.24.7.js.gz');
+            miSdkApiTag.setAttribute('src', 'https://folia-customer.s3.eu-west-1.amazonaws.com/mapsindoors/js/sdk/CandidateReleases/4.25.0-rc0/mapsindoors-4.25.0-rc0.js.gz');
```

Comment:
Remember to update this.

We can resolve this comment when done so we don't accidentally merge it 😄